### PR TITLE
cleanup launchers 

### PR DESCRIPTION
**Pull request by ebarboni**
removing unused Make and related
removing netbeans project file

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,10 +21,10 @@ under the License.
     <parent>
         <groupId>org.apache.netbeans</groupId>
         <artifactId>netbeans-parent</artifactId>
-        <version>2</version>
+        <version>3</version>
     </parent>
     <groupId>org.apache.netbeans.native</groupId>
-    <version>12.6-SNAPSHOT</version>
+    <version>15.0-SNAPSHOT</version>
```
Comment (neilcsmith-net) [inline pom.xml:27]: If launchers are released separately, they should probably be versioned separately and differently to the IDE.

I'd go with `13.0-SNAPSHOT` or `2022.0-SNAPSHOT` - possibly the latter.

**Comment by neilcsmith-net** [comment]
I'm probably not the best person to review this at the moment - makes sense, but haven't tried building for a long time - if it builds, it builds!

File: pom.xml (hunk 1 above)
Comment (ebarboni) [reply pom.xml]: using 2022-11 to have an idea of time and keep incrementing

**Comment by ebarboni** [comment]
it builds,

**Pull request merged**